tox.ini

```diff
@@ -79,8 +79,7 @@ deps =
     {tests,docstrings,checkdocstrings,analyze}: hypothesis
     lint: flake8
     {format,checkformatting}: black
-    {format,checkformatting}: six
-    {format,checkformatting}: isort[requirements]
+    {format,checkformatting}: isort
```

Comment:
Because the default is 3rd party, we don't need to read the requirements file (which is what this does)

Reply:
Yeah. It really is a better default